### PR TITLE
Improve documentation of influence package:

### PR DESCRIPTION
**Pull request by schroedk**
* add more information
* change to google style format

### Description

Google style docstrings for influence package

### Checklist

- ~~[ ] Wrote Unit tests (if necessary)~~
- [x] Updated Documentation (if necessary)
- ~~[ ] Updated Changelog~~
- ~~[ ] If notebooks were added/changed, added boilerplate cells are tagged with `"nbsphinx":"hidden"`~~

### CODE REVIEW

File: src/pydvl/influence/__init__.py [hunk 1]
```diff
@@ -1,10 +1,8 @@
 """
 This package contains algorithms for the computation of the influence function.
 
-.. warning::
-   Much of the code in this package is experimental or untested and is subject
-   to modification. In particular, the package structure and basic API will
-   probably change.
+> **Warning:** Much of the code in this package is experimental or untested and is subject to modification.
```
Comment (AnesBenmerzoug) [inline src/pydvl/influence/__init__.py:4]: Do we still need this warning even after the refactoring?

Comment (schroedk) [reply src/pydvl/influence/__init__.py:4]: I would like to keep it for now.

**Comment by schroedk** [comment]
Added reference to pytorch objects

**Comment by schroedk** [comment]
> @schroedk mkdocstrings has support for referencing other project's documentation using [inventories](https://mkdocstrings.github.io/usage/?h=inventory#cross-references-to-other-projects-inventories). Here's the for PyTorch: https://pytorch.org/docs/stable/objects.inv. I think we should add that.
> 
> Did you manage to find a fix for the references inside docstrings?

Added reference to pytorch objects.
It was only possible to insert the references directly into the python files.

**Pull request merged**